contents/handbook/company/management.md

```diff
@@ -15,12 +21,16 @@ A manager at PostHog has two tasks:
 That's it.
 
 A manager at PostHog is _not_ responsible for:
-1. Setting compensation (we have transparent compensation)
+1. Setting compensation (we have a [compensation calculator](/handbook/people/compensation) and increases are done occasionally by the people team).
 1. Setting tasks for their direct reports
 1. Creating a career path (career paths should be transparent and documented, and for now centrally managed)
 1. "Approving," whether that's projects, expenses, days off or accounts (people should have admin access by default to most things)
 1. Giving feedback (managers give feedback in their capacity as individual contributor, but so does everyone else)
 
+## How do I make sure my direct reports are happy and productive?
```

Comment:
Add point around checking in on workload and mental health occassionally

Reply:
Some of that is covered in the 1:1 doc, maybe worth expanding that?